### ISSUE TITLE
Is it possible to run this model on multiple smaller segments generated by Silero VAD or for streaming?

### ISSUE THREAD

**Issue by Fannovel16** [khanld/chunkformer#1]

**Comment by Fannovel16** [comment]
I figured it out, just writing the audio segment paths to [audio_list.tsv](https://github.com/khanld/chunkformer/blob/main/data/audio_list.tsv) file

**Comment by Fannovel16** [comment]
Wow this ASR is so fast, processes 2h of audio in only 8 seconds

**Comment by khanld** [comment]
Yup, I'm lazy, so I want a model to handle everything, like batching and padding with no cost, extremely low memory cost, and fast, but still good. So, I just simply put all the audio into the model and chilled.

**Comment by Fannovel16** [comment]
@khanld Can you convert the model to ONNX? It's more optimized to CPU and intermediate step to edge devices

**Comment by khanld** [comment]
I tried once, but Chunkformer has used some advanced ops that ONNX doesn't support yet, it's quite challenging to convert it

**Comment by khanld** [comment]
Also, I don't think the current public pre-trained checkpoints are good enough for application, since all the training data originate from YouTube (Vietnamese) or reading datasets (English)
